### ISSUE TITLE
MacOSX 11 build

### ISSUE THREAD

**Issue by robotaiguy** [AD-Vega/qarv#25]
Has anyone gotten this to work on OSX 11?  I get to the cmake .. step. Anybody have any idea?

(base) suneberg@Robobook build % cmake ..             
Program version not specified, using git description.
QArv version: '4466049'
-- Checking for one of the modules 'aravis-0.8;aravis-0.6;aravis-0.4;aravis-0.2;aravis'
CMake Error at /usr/local/Cellar/cmake/3.20.3/share/cmake/Modules/FindPkgConfig.cmake:843 (message):
  None of the required 'aravis-0.8;aravis-0.6;aravis-0.4;aravis-0.2;aravis'
  found
Call Stack (most recent call first):
  CMakeLists.txt:46 (pkg_search_module)

-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- Configuring incomplete, errors occurred!
See also "/Users/suneberg/Git/qarv/build/CMakeFiles/CMakeOutput.log".
See also "/Users/suneberg/Git/qarv/build/CMakeFiles/CMakeError.log".

From CmakeError.log:
Compiling the C compiler identification source file "CMakeCCompilerId.c" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc 
Build flags: 
Id flags:  

The output was:
1
ld: library not found for -lSystem
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ 
Build flags: 
Id flags:  

The output was:
1
ld: library not found for -lc++
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by exzombie** [comment]
Personally, I never used OSX, and I have no experience with XCode toolchain. How did you install Aravis? Apparently, `pkg_config` can't find it. Or anything else, for that matter.

@shrx any ideas? Could homebrew help?

**Comment by shrx** [comment]
Yeah, it looks like the aravis libraries are not found. homebrew has an [aravis](https://formulae.brew.sh/formula/aravis) formula, so I'd try using that and compiling with homebrew-provided gcc.
